features/admin.applications.v1/pages/applications.tsx

```diff
@@ -269,10 +341,12 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
             return;
         }
 
-        if (myAccountStatusFetchRequestError.response
-            && myAccountStatusFetchRequestError.response.data
-            && myAccountStatusFetchRequestError.response.data.description) {
-            if (myAccountStatusFetchRequestError.response.status === 404) {
+        if (
+            myAccountStatusFetchRequestError?.response
+            && myAccountStatusFetchRequestError?.response.data
+            && myAccountStatusFetchRequestError?.response.data.description
+        ) {
+            if (myAccountStatusFetchRequestError?.response.status === 404) {
```

Comment:
This `?` is not necessary since we already do a null check in the initial `if` condition.

Reply:
Fixed with be2f72f7a51adfae24c45849553614a1f98b8365.